module/VuFind/src/VuFind/Search/SolrAlternative/Results.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Search Results for second Solr index
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Staats- und Universitätsbibliothek Hamburg 2018.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Search_SolrAlternative
+ * @author   Hajo Seng <hajo.seng@sub.uni-hamburg.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+namespace VuFind\Search\SolrAlternative;
+
+/**
+ * Search Results for second Solr index
+ *
+ * @category VuFind
+ * @package  Search_SolrAlternative
+ * @author   Hajo Seng <hajo.seng@sub.uni-hamburg.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+class Results extends \VuFind\Search\Solr\Results
+{
+    /**
+     * Support method for performAndProcessSearch -- perform a search based on the
+     * parameters passed to the object.
+     *
+     * @return void
+     */
+    protected function performSearch()
```

Comment:
Rather than overriding this function, it might be even simpler to simply override the backendId property... i.e.:

<pre>
/**
 * Backend ID
 *
 * @var string
 */
protected $backendId = 'SolrAlternative';
</pre>

...at least, I'm pretty sure that should work. That way you ensure that the property is consistently set to the right value in case other methods also rely on it.